docs/source/contributing/releasing.adoc

```diff
@@ -28,8 +21,38 @@ defined in your environment as _GITHUB_ACCESS_TOKEN_.
   For example, issues with a merged pull requests should be labeled as *resolution/done*.
 . Close milestone.
 
+[[auto-relase]]
+== Automated Release
+
+An automated release can be performed as:
+
+----
+$ export API_KEY=<api-key>
+$ export RELEASE_VERSION=<expected release version> # Eg : 1.0.0 (without 'v')
```

Comment:
We can also get the few lines of Makefile script from downstream which can check whether a variable is set and if not aborts with an error message. This way we can abort the `make start-co-release` if the required variables are not set. 

As another positive side-effect we will have less downstream customisations.